Formula/qemu.rb

```diff
@@ -1,10 +1,9 @@
 class Qemu < Formula
   desc "Emulator for x86 and PowerPC"
   homepage "https://www.qemu.org/"
-  url "https://download.qemu.org/qemu-6.2.0.tar.xz"
-  sha256 "68e15d8e45ac56326e0b9a4afa8b49a3dfe8aba3488221d098c84698bca65b45"
+  url "https://download.qemu.org/qemu-7.0.0.tar.xz"
```

Comment:
QEMU 7.0 is known to cause host kernel panic on M1 (Max and Pro only?) version of macOS 12.3 when the guest RAM exceeds 4 GiB. The kernel panic is reported to be fixed in macOS 12.4 Beta: https://github.com/lima-vm/lima/issues/795

Maybe better to hold this PR until the release of macOS 12.4?
Or at least it should print a caveat message?

Reply:
Thank you for the information, maybe we should upgrade to 7.0.0 after macOS 12.4 released.

Reply:
according to https://gitlab.com/qemu-project/qemu/-/issues/903#note_876417804 this also happens with the current version in homebrew - so it should not block 7.0.0

Reply:
According to https://github.com/lima-vm/lima/pull/796#issuecomment-1104026408 https://github.com/lima-vm/lima/pull/796#issuecomment-1104041403 , 7.0.0 crashes but 6.2.1 does not,

I guess https://gitlab.com/qemu-project/qemu/-/issues/903#note_876417804 is hitting a different code path.

Reply:
I think its "6.2.0 with homebrew patches" that crashes. homebrew contains several backported 7.0.0 patches

Reply:
https://github.com/lima-vm/lima/pull/796#issuecomment-1104026408 says “ brew 6.2.0_1” does not crash.

Probably https://gitlab.com/qemu-project/qemu/-/issues/903#note_876417804 is about slightly different command line options?

Reply:
sorry, 6.2.0_1 is brew. seems you are right.